apps/site/assets/ts/stop/components/amenities/ParkingAmenityCard.tsx

```diff
@@ -117,18 +119,71 @@ const getModalContent = (
   );
 };
 
+const getBadge = (
+  parkingLots: ParkingLot[],
+  areAllParkingLotsClosed: boolean
+): JSX.Element => {
+  if (parkingLots.length === 0) {
+    return <Badge text="Not available" bgClass="u-bg--gray-lighter" />;
+  }
+  if (areAllParkingLotsClosed) {
+    return <Badge text="Temporarily closed" />;
+  }
+  return <></>;
+};
+
+const allParkingLotsClosed = (
+  alerts: Alert[],
+  parkingLots: ParkingLot[]
+): boolean => {
+  if (parkingLots.length === 0) {
+    return false;
+  }
+  const parkingClosuresIDs = alertsForEffect(alerts, "parking_closure").flatMap(
+    a => a.informed_entity.facility
+  );
+  const parkingLotIDs = parkingLots.map(p => p.id);
+
+  // if both arrays have the same ids in them then all lots are closed
+  // meaning `openParkingLotIDs` should be empty if all lots are closed
+  const openParkingLotIDs = difference(parkingLotIDs, parkingClosuresIDs);
+
+  return openParkingLotIDs.length === 0;
+};
+
 const ParkingAmenityCard = ({
   stop,
   alertsForParking
 }: {
   stop: Stop;
   alertsForParking: Alert[];
 }): JSX.Element => {
-  const icon = <span className="m-stop-page__icon">{parkingIcon()}</span>;
+  const icon = (
+    <span className="m-stop-page__icon u-color-brand-primary">
+      {parkingIcon()}
+    </span>
+  );
   const modalContent = getModalContent(stop, alertsForParking);
+  const allParkingClosed = allParkingLotsClosed(
+    alertsForParking,
+    stop.parking_lots
+  );
+  const badge = getBadge(stop.parking_lots, allParkingClosed);
 
   return (
-    <AmenityCard headerText="Parking" icon={icon} modalContent={modalContent} />
+    <AmenityCard
+      headerText="Parking"
+      icon={icon}
+      modalContent={modalContent}
+      badge={badge}
+    >
+      {stop.parking_lots.length === 0 && (
+        <div>This station does not have parking</div>
+      )}
+      {!allParkingClosed && stop.parking_lots.length !== 0 && (
+        <div>View daily rates and facility information</div>
+      )}
```

Comment:
I wonder if there needs to be something to show here when only some parking lots are closed.

Reply:
If only some of the lots are closed the `View daily rates and facility information` should be showing

Reply:
Ah right of course!